docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -22,19 +22,27 @@ To connect to Databricks Delta Lake, you need:
 - Network access to the HMS from the coordinator and workers. Port 9083 is the
   default port for the Thrift protocol used by the HMS.
 - Data files stored in the [Parquet file format](parquet-format-configuration)
-  on a [supported file system](delta-lake-file-system-configuration).
+  on a [supported file system](delta-lake-file-system-configuration). You must
+  select and configure the desired file system in your catalog `.properties`
+  file.
```

Comment:
This should be a separate bullet.

Reply:
"in your catalog configuration" reads better to me than "in your catalog .properties file"